### PR TITLE
Fixes #17483 - async task creation returns 202

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by shlomizadok** [comment]
++ LGTM

File: app/controllers/api/v2/reports_controller.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@ def show
       def create
         import_params = { :report => params[:report] }
         task = ForemanTasks.async_task Actions::Foreman::Report::Import, import_params, resource_class, detected_proxy.try(:id)
-        process_success _('Report import has been enqueued, please see the task %s for more details') % task.id
+        process_success _('Report import has been enqueued, please see the task %s for more details') % task.id, :accepted
```
Comment (dLobatog) [inline app/controllers/api/v2/reports_controller.rb]: bonus points by @ekohl (he promised it on the ticket 😄) if you provide the URL along with the task ID

**Review by dLobatog — CHANGES_REQUESTED** [review]
@ares Pretty much 👍  - just that it'd be nice to get the task URL to check the status in 1 click, the ID is helpful too but it's less friendly

Thanks!!!

**Comment by ares** [comment]
Unfortunately the `process_success` can't simply render the string, since it uses `respond_with`. I added the link to he message which is now included in the response. Keeping it in separate commit, ping me if you want me to squash it. In future we might standardize such messages in the whole API, but I guess separately from this issue.

**Review by dLobatog — APPROVED** [review]

**Comment by dLobatog** [comment]
@ares Thank you! I do agree with the standardization, one step at a time. I'll squash it on merge if tests pass 🎉

**Comment by ekohl** [comment]
@ares remind me I owe you a beer. FOSDEM or CfgMgmtCamp? :)

**Comment by ares** [comment]
Hehe, I hope to see you there to pay my debts too :-)

**Pull request merged**